core/templates/core/announcement_email.html

```diff
@@ -0,0 +1,9 @@
+{% autoescape off %}
+Below is an announcement from {{user}}:
+
+{{body}}
+
+Best,
+{{user}}
```

Comment:
Also, announcements have a target field that lets the user add a link, I think we should add after the body a paragraph that says `{{ user }} included a link in their announcement, click <a href="{{ target }}">here</a> to view`.  Also maybe a line with a link to the index page where the announcement is that says like `Click here to view this announcement on greek-rho.com` or something like that.  Idk that might be too hard